tests/test_model.py

```diff
@@ -3219,7 +3219,30 @@ def test_update_external_index(self):
         # The work was not added to the search index -- that happens
         # later, when the WorkCoverageRecord is processed.
         eq_([], index.docs.values())
-        
+      
+
+    def test_for_unchecked_subjects(self):
+
+        w1 = self._work(with_license_pool=True)
+        w2 = self._work()
+        identifier = w1.license_pools[0].identifier
+
+        # Neither of these works is associated with any subjects, so
+        # they're not associated with any unchecked subjects.
+        qu = Work.for_unchecked_subjects(self._db)
+        eq_([], qu.all())
+
+        # This Subject hasn't been checked, so the Work associated with
+        # it shows up.
+        ds = DataSource.lookup(self._db, DataSource.OVERDRIVE)
+        classification = identifier.classify(ds, Subject.TAG, "some tag")
+        classification2 = identifier.classify(ds, Subject.TAG, "another tag")
+        eq_([w1], qu.all())
+
+        # Mark it as checked, and the Work stops showing up.
+        classification.subject.checked = True
```

Comment:
I don't understand why you don't also have to mark classification2 as checked, since the identifier is under both.

Reply:
Maybe this is related to why my tests are failing... :|

Reply:
I'd like to see the test check that the work still shows up when only one subject is marked as checked, if that's what's supposed to happen.